LICENSE.md

```diff
@@ -1,7 +1,6 @@
 MIT License
 
-Copyright (c) 2018-2022 Anthony De Meulemeester (@anthdm), City of Zion
-community (@CityOfZion), NeoSPCC (@nspcc-dev)
+Copyright (c) 2018-2023 Anthony De Meulemeester (@anthdm), City of Zion community (@CityOfZion), NeoSPCC (@nspcc-dev)
```

Comment:
Maybe it's time to sort them by contribution? NeoSPCC first, Anthony second and COZ last?

Reply:
Fixed.